app/src/main/java/org/p2p/wallet/newsend/ui/details/NewSendDetailsBottomSheet.kt

```diff
@@ -136,7 +137,7 @@ class NewSendDetailsBottomSheet :
                     setTextColor(colorNight)
                     if (fee == null) {
                         val zeroUsd = "(0$)"
-                        val fullZeroFeeText = "0 ${state.sourceSymbol} $zeroUsd"
+                        val fullZeroFeeText = "0 ${Constants.SOL_SYMBOL} $zeroUsd"
```

Comment:
no, that's not correct!

Reply:
https://p2pvalidator.atlassian.net/browse/PWN-6616?focusedCommentId=41057

@1sakov ^^

Reply:
i am on maximum XZ here
but they want it =)